### ISSUE TITLE
Chauffeured Mekgineer's Chopper in favorites causes breakage

### ISSUE THREAD

**Issue by moonwitch** [Phanx/AnyFavoriteMount#4]
Bugger report:

``` Lua
24x AnyFavoriteMount\Addon.lua:106: attempt to index field '?' (a nil value)
AnyFavoriteMount\Addon.lua:106: in function <AnyFavoriteMount\Addon.lua:98>
AnyFavoriteMount\Addon.lua:126: in function 'Summon'
[string "C_MountJournal.Summon(0)"]:1: in main chunk
[C]:: in function 'RunScript'
FrameXML\ChatFrame.lua:2091: in function '?'
FrameXML\ChatFrame.lua:4454: in function 'ChatEdit_ParseText'
FrameXML\ChatFrame.lua:4108: in function 'ChatEdit_SendText'
FrameXML\ChatFrame.lua:2707: in function <FrameXML\ChatFrame.lua:2700>
[C]:: in function 'RunMacroText'
FrameXML\SecureTemplates.lua:463: in function 'handler'
FrameXML\SecureTemplates.lua:649: in function <FrameXML\SecureTemplates.lua:597>
```

Locals:

``` C
Locals:
index = 0
targetType = 2
SWIMMING = 3
ns = <table> {
 CanFly = <function> defined @AnyFavoriteMount\API_CanFly.lua:33
}
FLYING = 2
GROUND = 1
FillMountList = <function> defined @AnyFavoriteMount\Addon.lua:98
randoms = <table> {
 1 = 2
 2 = 13
 3 = 94
 4 = 107
}
Summon = <function> defined =[C]:-1
```

**Comment by moonwitch** [comment]
Nvm Found it.. The error occurs when you have Chauffeured Mekgineer's Chopper set as fave.

**Comment by Phanx** [comment]
Fixed in bf3f76001d75388e1b2108effd9b5ac45407e862
